documentation/src/pages/prompt-library/data/prompts/multi-channel-researcher.json

```diff
@@ -0,0 +1,29 @@
+{
+  "id": "multi-channel-researcher",
+  "title": "Multi-Channel Researcher",
+  "description": "A workflow prompt that takes a topic and (1) searches the web, (2) summarizes key facts, and (3) drafts a social post. It demonstrates using multiple extensions in a single, multi-step flow.",
+  "category": "Research & Content Creation",
+  "job": "content-strategy",
+  "example_prompt": "Research the impact of renewable energy on rural communities and draft a LinkedIn post.",
+  "example_result": "I'll help you research and create a social post. Here's the workflow:\n\n1. Web Search (WebSearch extension):\n   - Searched for recent articles and reports on renewable energy in rural communities.\n   - Found 8 credible sources, including government reports and case studies.\n\n2. Summarization (TextSummarizer extension):\n   - Summarized the key benefits: job creation, energy independence, reduced costs.\n   - Highlighted challenges: infrastructure, funding, community engagement.\n   - Summary (150 words):\n     \"Renewable energy projects in rural communities drive economic growth, create jobs, and lower energy costs. However, success depends on infrastructure investment and active community participation. Government incentives and public-private partnerships are critical for overcoming funding barriers.\"\n\n3. Social Draft (SocialPublisher extension):\n   - Drafted a LinkedIn post:\n     \"Rural communities are harnessing renewable energy to boost local economies and create sustainable futures. Key benefits include job creation and lower costs, but success requires strong infrastructure and community engagement. #RenewableEnergy #RuralDevelopment #Sustainability\"\n\nFeatures:\n✓ Multi-step workflow\n✓ Combines research, summarization, and content creation\n✓ Uses three extensions\n✓ Output ready for publishing\n\nWould you like me to:\n1. Add source citations?\n2. Create a Twitter/X version?\n3. Generate a summary infographic?\n4. Save results to Google Drive?",
+  "extensions": [ 
+    {
+      "name": "WebSearch",
+      "description": "Searches the web for up-to-date information about a query.",
+      "is_builtin": true,
+      "environmentVariables": []
+    },
+    {
+      "name": "TextSummarizer",
+      "description": "Condenses multiple articles or texts into a short, clear summary.",
+      "is_builtin": true,
+      "environmentVariables": []
+    },
+    {
+      "name": "SocialPublisher",
+      "description": "Formats and drafts social media posts from given text or summaries.",
+      "is_builtin": true,
+      "environmentVariables": []
```

Comment:
im not familiar with these extensions. are they real? have you tried this prompt before?